E621Client/Area/Post/E621Client.cs

```diff
@@ -10,113 +10,19 @@ namespace Noppes.E621
-        /// position parameter defines the position of the returned posts relative to the given post
-        /// ID. Let's take post with ID 1000 as an example. Passing this ID in combination with <see
-        /// cref="Position.Before"/> will cause the posts 999, 998, 997, etc. to be retrieved. Using
-        /// <see cref="Position.After"/> will retrieve the posts 1001, 1002, 1003, etc. You should
-        /// use this method if you don't need pagination or need to avoid the limit pagination
-        /// comes with it. Moreover, this is the most efficient way to navigate through posts.
-        /// </para>
-        /// </summary>
-        /// <param name="tags">Tags used to filter the result.</param>
-        /// <param name="limit">The maximum number of posts returned in a single request.</param>
-        /// <param name="id">ID of the post you want the retrieved posts the be relatively positioned to.</param>
-        /// <param name="position">Relative position to the given post ID.</param>
-        /// <exception cref="ArgumentException"></exception>
-        /// <exception cref="ArgumentOutOfRangeException"></exception>
+        /// <inheritdoc cref="IE621Client.GetPostsAsync(int,Noppes.E621.Position,System.Nullable{int},string?)"/>
         public Task<ICollection<Post>> GetPostsAsync(int id, Position position, int? limit = null, string? tags = null) => GetPostsAsync(tags, limit, id, position);
 
-        /// <summary>
-        /// Retrieves a collection of posts based on the given parameters.
-        /// <para>
-        /// You can choose which posts to show and hide by filtering on tags. The maximum number of
-        /// tags that can be used, is defined at <see cref="PostsMaximumTagSearchCount"/>. Exceeding
-        /// this limit, will cause an exception to be thrown.
-        /// </para>
-        /// <para>
-        /// Be aware that there is a limit on the number of posts which can be retrieved with a
-        /// single call. This limit is defined at <see cref="PostsMaximumLimit"/>. Exceeding this
-        /// limit, will cause an exception to be thrown.
-        /// </para>
-        /// <para>
-        /// You can navigate through all the posts by specifying a page number. There is, however, a
-        /// limit to the value. This limit is defined in <see cref="PostsMaximumPage"/>. Exceeding
-        /// this limit, will cause an exception to be thrown.
-        /// </para>
-        /// </summary>
-        /// <param name="tags">Tags used to filter the result.</param>
-        /// <param name="limit">The maximum number of posts returned in a single request.</param>
-        /// <param name="page">Pagination, number of the page.</param>
-        /// <exception cref="ArgumentException"></exception>
-        /// <exception cref="ArgumentOutOfRangeException"></exception>
+        /// <inheritdoc cref="IE621Client.GetPostsAsync(int,Noppes.E621.Position,System.Nullable{int},string?)"/>
```

Comment:
The `cref` attribute points to `IE621Client.GetPostsAsync(int,Noppes.E621.Position,System.Nullable{int},string?)``, but the method starts with a parameter of type `string?`. The reference seems to be pointing to the wrong method?

Reply:
I had some trouble changing this at first but now i've figured it out.